examples/avrami_method_example.py

```
@@ -1,5 +1,5 @@
 import numpy as np
-from model_fitting_methods import avrami_method, avrami_equation, calculate_half_time
+from model_fitting_methods import jmak_method, jmak_equation, calculate_half_time
```

Ruff

`model_fitting_methods.jmak_method` imported but unused

Remove unused import

(F401)